server/database/dbOperations.ts

```diff
@@ -229,3 +239,397 @@ export const removeTableFromConfig = async (
     throw error;
   }
 };
+
+// Annotated Collections Operations
+
+export const createAnnotatedCollection = async (
+  db: DatabaseConnection,
+  collection: Omit<AnnotatedCollection, "id" | "created_at" | "updated_at">,
+  incidentData?: Omit<Incident, "collection_id">,
+  entries?: Array<{
+    source_table: string;
+    source_id: string;
+    notes?: string;
+  }>,
+): Promise<AnnotatedCollection> => {
+  const pgClient = db as Client;
```

Comment:
Can we add a docstring? Thinking back to the discussion we had in https://github.com/ConservationMetrics/gc-landing-page/pull/30.